### ISSUE TITLE
[LOW] [P2P] Wallet - User is unable to transfer funds from wallet

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#35007]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.30-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @zsgreenwald 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1705436286459439

## Action Performed:
1. Navigate to Wallet 
2. Try Transfer funds from wallet using any of the options

## Expected Result:
User can transfer funds from wallet successfully
 
## Actual Result:

- 1-3 business days option returned me `Missing Translation`
- Instant transfer option shows an endless spinner

## Workaround:
Unknown. 

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

![image](https://github.com/Expensify/App/assets/44479856/c61ecf59-d672-4766-b8ec-0c6a1c2eb7c6)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0150061b180d05b0b4</li>
        <li>Upwork Job ID: 1749908512847069184</li>
        <li>Last Price Increase: 2024-01-23</li>
    </ul>
</details>

**Comment by isagoico** [comment]
Since this involves wallet and transferring funds I will add the "Internal" label. Please correct me if this was not the correct label 😅

**Comment by getusha** [comment]
@slafortune I can try to pull internal engineer if needed.

**Comment by slafortune** [comment]
@getusha since the hotpick tag and Project is assigned, that will indicate to an internal engineer their help is needed.

**Comment by slafortune** [comment]
still waiting on someone to pick this up

**Comment by slafortune** [comment]
All good Melvin - low priority for the time being

**Comment by slafortune** [comment]
Still a low priority - no one picked up yet

**Comment by slafortune** [comment]
Still waiting on an internal engineer - was posted in the channel updates [here](https://expensify.slack.com/archives/C05RECHFBEW/p1708562734937009)

**Comment by slafortune** [comment]
Still waiting on an internal engineer

**Comment by slafortune** [comment]
Labeled HotPick - waiting on an internal engineer

**Comment by mvtglobally** [comment]
QA team is not able to verify this issue during internal KI retests as this required production KYC passed accounts.

**Comment by tgolen** [comment]
I confirmed it works. Thanks for the dollar @bondydaa !

https://github.com/Expensify/App/assets/1228807/99e0abc4-5075-4c1e-b252-e3a3600038d9
